.github/workflows/release.yml

```diff
@@ -209,13 +210,14 @@ jobs:
 
       - name: Build
         run: |
+          brew install molten-vk
```

Comment:
Should we add `brew uninstall molten-vk` after we complete the building process to make sure the test can pass without molten-vk?

Reply:
Sure, if it does not take a lot of time